### DISCUSSION TITLE
Vulnerable moment.js version in PrimeFaces 10.x branch

### DISCUSSION THREAD

**Discussion by chovyy** [primefaces/community#301] [PrimeFaces] [answered]
@PrimeTek: Last year, you updated the moment.js version in the 11.x and 12.x branch due to CVE-2022-31129 (see primefaces/primefaces#8968), but the 10.x branch is still vulnerable. Can you update the latter as well?

**Comment by melloware** [comment, marked as answer]
@chovyy see https://github.com/primefaces/primefaces#community--elite--pro and request it through your PRO account.

**Comment by chovyy** [comment]
I'm sorry, I would have expected that ~you~ _PrimeTek_ provide security patches for all ~your~ _their_ paying users including ELITE.

**Reply by melloware** [reply]
I don't work for PrimeTek. I am just an open source community member

**Reply by chovyy** [reply]
I'm sorry. With "you" I didn't mean specifically you, @melloware .

**Reply by chovyy** [reply]
Nevertheless, the whole thing here is totally confusing. Due to https://github.com/primefaces/primefaces#community--elite--pro I haven't opened a regular issue, but started the discussion here, since that is the official replacement for https://forum.primefaces.org/.  

I know, that the ELITE FAQ says: "Does ELITE enable us to contact PrimeFaces Team to receive support? No, that is provided by PRO support.", but I can expect that they respond to questions regarding security issues on their official community forum. Am I wrong?

**Reply by melloware** [reply]
Yeah you can post here but PrimeTek has said this is community forums... as is Discord, its not an official support channel. It is for community members to support other members.  So PRO is your only way to have a direct contact with PrimeTek.
